docs/configuration.txt

```diff
@@ -115,6 +115,29 @@ Translog
 
   Disable/enable flushing.
 
+Cache
+-----
+
+**index.cache.filter.max_size**
+  | *Default:*    ``5``
+  | *Runtime:*   ``yes``
+
+The maximum size of filter cache (per segment in shard). Set to -1 to disable.
+
+Gateway
+-------
+
+**index.gateway.local.sync**
+  | *Default:*    ``5``
+  | *Runtime:*   ``yes``
+
+  Sets the period of translog fsyncing to the disk
+
+  .. note::
+
+  Keep in mind that data which was not ``fsync``ed to the disk can be lost in case of a power failure.
```

Comment:
please add more detailed information about this setting according to https://www.elastic.co/guide/en/elasticsearch/guide/master/translog.html as we discussed in the daily